File: explorer-fe-main/src/app/modules/tours/interest.ts
export class Interest {
  id: number;
  interestTypeName: number;
  touristId: number;
  tourId: number;

  constructor(interestTypeName: number, touristId: number, tourId: number, id: number) {
    this.interestTypeName = interestTypeName;
    this.touristId = touristId;
    this.tourId = tourId;
    this.id = id;
  }
}
export enum InterestType {
  nature, art, sport, shopping, food
}

File: explorer-be-main/src/HospitalLibrary/Core/Model/Interest.cs
using HospitalLibrary.Core.Model.Enum;

namespace HospitalLibrary.Core.Model
{
    public class Interest : Entity
    {
        //u db contexu napuni primerima sa trello Model kartice
        public InterestType InterestTypeName { get; set; }
        public int TouristId { get; set; }
        public int TourId { get; set; }
    }
}